emulator.h

```diff
@@ -69,6 +77,12 @@ typedef struct
   bool interrupt_enabled;
   bool halted;
 
+  bool colored_screen;
```

Comment:
I guess we can leave this in for now though not sure we'll get to it. Don't need to remove for my approval but would be good to take this out if we don't get to it.